code/modules/spells/roguetown/wizard.dm

```diff
@@ -1154,6 +1240,494 @@ Please whenever possible, make each spell its own procholder, and do *not* have
+	chargedloop = /datum/looping_sound/invokegen
+	associated_skill = /datum/skill/magic/arcane //can be arcane, druidic, blood, holy
+	cost = 3
+
+	xp_gain = TRUE
+	miracle = FALSE
+
+/obj/projectile/magic/frostbolt
+	name = "Frost Dart"
+	icon_state = "ice_2"
+	damage = 20
+	nodamage = FALSE
+	damage_type = BURN
+	flag = "magic"
+	range = 10
+	speed = 8 //higher is slower
+	var/aoe_range = 0
+
+/obj/projectile/magic/frostbolt/on_hit(target)
+	. = ..()
+	if(ismob(target))
+		var/mob/M = target
+		if(M.anti_magic_check())
+			visible_message(span_warning("[src] fizzles on contact with [target]!"))
+			playsound(get_turf(target), 'sound/magic/magic_nulled.ogg', 100)
+			qdel(src)
+			return BULLET_ACT_BLOCK
+		if(isliving(target))
+			var/mob/living/L = target
+			L.apply_status_effect(/datum/status_effect/buff/frostbite)
+			new /obj/effect/temp_visual/snap_freeze(get_turf(L))
+//	qdel(src)
+
+/obj/effect/temp_visual/snap_freeze
+	icon = 'icons/effects/effects.dmi'
+	icon_state = "shieldsparkles"
+	name = "rippeling arcyne ice"
+	desc = "Get out of the way!"
+	randomdir = FALSE
+	duration = 1 SECONDS
+	layer = MASSIVE_OBJ_LAYER
+
+/obj/effect/proc_holder/spell/targeted/lightninglure
+	name = "Lightning Lure"
+	desc = "An electric connection forms between you and the target, and after several seconds of build up, shocks the target if they remain nearby."
+	overlay_state = "null"
+	releasedrain = 50
+	chargetime = 1
+	charge_max = 12 SECONDS
+	range = 3
+	warnie = "spellwarning"
+	movement_interrupt = FALSE
+	no_early_release = FALSE
+	chargedloop = null
+	sound = 'sound/magic/whiteflame.ogg'
+	chargedloop = /datum/looping_sound/invokegen
+	associated_skill = /datum/skill/magic/arcane //can be arcane, druidic, blood, holy
+	cost = 3 // might even deserve a cost of 3
+
+	xp_gain = TRUE
+	miracle = FALSE
+
+	invocation = ""
+	invocation_type = "shout" //can be none, whisper, emote and shout
+	include_user = FALSE
+
+	var/delay = 5 SECONDS
+	var/sprite_changes = 10
+	var/datum/beam/current_beam = null
+
+/obj/effect/proc_holder/spell/targeted/lightninglure/cast(list/targets, mob/user = usr)
+	for(var/mob/living/carbon/C in targets)
+		user.visible_message(span_warning("[C] is connected to [user] with a lightning lure!"), span_warning("You create a static link with [C]."))
+		playsound(user, 'sound/items/stunmace_gen (2).ogg', 100)
+
+		var/x
+		for(x=1; x < sprite_changes; x++)
+			current_beam = new(user,C,time=50/sprite_changes,beam_icon_state="lightning[rand(1,12)]",btype=/obj/effect/ebeam, maxdistance=10)
+			INVOKE_ASYNC(current_beam, TYPE_PROC_REF(/datum/beam, Start))
+			sleep(delay/sprite_changes)
+
+		var/dist = get_dist(user, C)
+		if (dist <= range)
+			C.electrocute_act(1, user) //just shock
+		else
+			playsound(user, 'sound/items/stunmace_toggle (3).ogg', 100)
+			user.visible_message(span_warning("The lightning lure fizzles out!"), span_warning("[C] is too far away!"))
+
+/obj/effect/proc_holder/spell/invoked/mending
+	name = "Mending"
```

Comment:
This spell super invalidates an important part of seamsters and smiths' jobs. I'd rather if it were omitted.